helm-charts/templates/data-plane/gateway-components/log-conf.yaml

```diff
@@ -692,7 +692,7 @@ data:
 
     [[pkg]]
     name = "github.com/wso2/product-microgateway/adapter/internal/adapter"
-    logLevel = "INFO" # LogLevels can be "DEBG", "FATL", "ERRO", "WARN", "INFO", "PANC"
+    logLevel = "DEBG" # LogLevels can be "DEBG", "FATL", "ERRO", "WARN", "INFO", "PANC"
```

Comment:
do we need DEBG enabled by default? seems you forgot to revert it back

Reply:
no, I added it intentionally since we are working on the initial minor release still, and we are frequently in need of adapter debug logs. So I thought it was ok to keep it in debug mode till the minor release.  @tharindu1st wdyt? is it ok to keep adapter debug logs in helm charts temporarily?

Reply:
+1. I'll revert this. Let's remove this as this is only beneficial from the developer's perspective and can be a burden in other cases.